src/test/java/com/vinylteam/vinyl/web/servlets/SignInServletTest.java

```diff
@@ -17,15 +20,28 @@
     private final UserService mockedUserService = mock(UserService.class);
     private final HttpServletRequest mockedHttpServletRequest = mock(HttpServletRequest.class);
     private final HttpServletResponse mockedHttpServletResponse = mock(HttpServletResponse.class);
+    private final PrintWriter printWriter = new PrintWriter(new StringWriter());
     private final SignInServlet signInServlet = new SignInServlet(mockedUserService);
 
     @Test
-    @DisplayName("Checks if all right methods are called and response has code set to 200 and redirected to /home " +
+    @DisplayName("Checks if the response code is 200 when loading the page /signIn")
+    void doGetTest() throws IOException{
+        //prepare
+        when(mockedHttpServletResponse.getWriter()).thenReturn(printWriter);
+        //when
+        signInServlet.doGet(mockedHttpServletRequest, mockedHttpServletResponse);
+        //then
+        verify(mockedHttpServletResponse).setStatus(HttpServletResponse.SC_OK);
```

Comment:
You can add one more row for checking: verify(mockedHttpServletResponse).getWriter();